cx-sig-charter-draft.md

```diff
@@ -0,0 +1,81 @@
+# Contributor Experience Special Interest Group Charter
+
+This charter uses the Roles and Organization Management outlined in [sig-governance](https://github.com/spinnaker/governance/blob/master/governance.md).
+
+## Scope
+
+The Contributor Experience Special Interest Group (SIG) is responsible for improving the experience of those who contribute upstream to the Spinnaker project. We do this by creating, and maintaining programs and processes that promote community health and reduce project friction while retiring those programs and processes that don't. Being conscientious of our contributor base is critical to scaling the project, growing the ecosystem, and helping the project succeed.
+
+We do this by listening - whether it’s through our roadshows to SIG meetings, surveys, data, or [GitHub issues](https://github.com/spinnaker/spinnaker/issues), we take in the feedback and turn it into our project list. We build a welcoming and inclusive community of contributors by giving them places to be heard and productive.
+
+### In scope
+
+#### Projects & Services
+
+-  In accordance with the code of conduct, establish policies, standards, and procedures for the use, moderation, and management of all public platforms officially used by the project, including but not limited to:
```

Comment:
According to this language, it sounds like you're implicitly looking for administrative access to these tools. Is that correct? Would like clarification if not. If so, how do you plan to manage access control to these systems?

Reply:
@robzienert some aspects of this access will be managed through automation that the contributor experience will build. This building phase will create groups to handle things. For example, slack automation. That will not be blanket access but rather controlled by a process that will be based on GitOps, much like we have in the Kubernetes community.

Reply:
Good question @robzienert. Public platform access control is technically managed by the Continuous Delivery Foundation. My observation is that an additional human interface with the CDF is needed in addition to what the CDF provides to facilitate sharing and make sure that access info is communicated to fairly empower contributors while. CDF can help with secure sharing.